### ISSUE TITLE
Cannot import an API when there is no environments defined in the api_params.yaml file

### ISSUE THREAD

**Issue by npamudika** [wso2/product-apim-tooling#499]
**Description:**
APIs cannot be imported when there is no environments defined in the api_params.yaml file as below.

```
deploy:
  import:
    update: true
    preserveProvider: false
```
When tried to import an API with the above api_params.yaml file, it gives the follwing error.

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x88 pc=0xccf738]
```

This is an regression issue happened due to the changes added to fix https://github.com/wso2/product-apim-tooling/issues/477 for the 3.2.x and master branches. The following line causes the issue, as it tries to get the mutialSSLCerts from a nil value when the environments are not defined.

```
if envParams.MutualSslCerts != nil {
		err = generateMutualSslCertificates(importPath, envParams, preserveProvider)
```

For the 2.x branch, the issue is not there as the code segment is different and the error is handled by the code itself.

**Comment by npamudika** [comment]
This can be solved by doing the envParams.MutualSslCerts related logic inside the else block as below.

```
if envParams == nil {
		utils.Logln(utils.LogPrefixInfo + "Using default values as the environment is not present in api_param.yaml file")
	} else {
		//If environment parameters are present in parameter file
		err = mergeAPI(importPath, envParams)
		if err != nil {
			return err
		}

		err = generateEndpointCertificates(importPath, envParams)
		if err != nil {
			return err
		}

		// generate certificates for mutualssl, only if the field is specified
		if envParams.MutualSslCerts != nil {
			err = generateMutualSslCertificates(importPath, envParams, preserveProvider)
			if err != nil {
				return err
			}
		}
	}
```

**Comment by npamudika** [comment]
As a workaround for the issue, define environments in the api_params.yaml file as below.

```
environments:
  - name: dev
    endpoints:
      production:
      sandbox:
deploy:
  import:
    update: true
    preserveProvider: false
```

**Comment by npamudika** [comment]
This will be fixed in the next 3.2.x patch release and the 3.x. major release of APICTL.
